### PR TITLE
improve clarity on setup and prerequisites documentation

### LINKED ISSUE #333: doc updates: provide detailed tool download commands
(1) For quick start and each sample, please provide a detailed example about how to download, unpack, and wire up the tool into one's PATH. For example use `curl -m 120 -fL https://github.com/oracle/weblogic-image-tool/releases/latest/download/imagetool.zip -o ./imagetool.zip` to get the latest built version of the tool, and call `unzip imagetool.zip` to unpack into `./weblogic-image-tool`, and finally update your PATH environment to include the full absolute path to the tools 'bin' directory.

(2) For the developer instructions, and the 'build tool from source' quick start instructions, please provide a detailed example about how to clone the source. For example `git clone https://github.com/oracle/weblogic-image-tool` (optionally with `--branch xxx`) downloads the source to `./weblogic-image-tool`. Plus provide the exact mvn command for building everything up to and including the `imagetool.zip`.  For example, `cd ./weblogic-image-tool; mvn clean package` generates `./imagetool/target/imagetool.zip`.

### PR DESCRIPTION
**Pull request by ddsharpe**
Fixes #333. Minor updates to Build from Source, Setup, and Prerequisite documentation pages.

### CODE REVIEW

**Review by rosemarymarano — COMMENTED** [review]
Minor edits

**Review by rosemarymarano — APPROVED** [review]
Perfect. Thank you.

File: documentation/1.10/content/quickstart/setup.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ draft: false
 weight: 2
 ---
 
-- Download and unzip the WIT release ZIP file to a desired location.
-- OPTIONALLY: You may build the project (`mvn clean package`) to create the ZIP installer in `./imagetool/target`.
+- Download the WIT release ZIP file to a desired location.
+  - The latest ZIP file can be found on the project [releases](https://github.com/oracle/weblogic-image-tool/releases) page.
```
Comment (tbarnes-us) [inline documentation/1.10/content/quickstart/setup.md:9]: The latest ZIP file --> The latest release Zip file and Zip files for earlier releases

File: documentation/1.10/content/quickstart/setup.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ draft: false
...
+  - Alternatively, you can download the ZIP file with cURL.
+    ```shell
+    curl -m 120 -fL https://github.com/oracle/weblogic-image-tool/releases/latest/download/imagetool.zip -o ./imagetool.zip
+    ```
```
Comment (tbarnes-us) [inline documentation/1.10/content/quickstart/setup.md:13]: 1) "the ZIP file" --> the latest ZIP file
2) it'd be helpful IMO to also teach how to get a specific version (assuming that is straight forward) 
3) would it help if the non-quickstart samples could point to quick start?

Comment (tbarnes-us) [reply documentation/1.10/content/quickstart/setup.md:13]: 4. the sample assumes image tool bin is in the PATH, but has no instructions for setting same

**Review by robertpatrick — APPROVED** [review]

File: documentation/1.10/content/quickstart/setup.md (hunk 2 above)
Comment (robertpatrick) [reply documentation/1.10/content/quickstart/setup.md:13]: > 4. the sample assumes image tool bin is in the PATH, but has no instructions for setting same

I think we can assume that the user knows how to add a directory to the PATH.

**Review by tbarnes-us — APPROVED** [review]

**Pull request merged**